tests/phpunit/tests/post/thumbnails.php

```diff
@@ -79,11 +80,28 @@ public function test_update_post_thumbnail_cache() {
 			)
 		);
 
+		// Test case where `$query1->posts` should return Array of post IDs.
+		$query1 = new WP_Query(
```

Comment:
Nit pick, let's use 2.
```suggestion
		$query2 = new WP_Query(
```